test/uk/gov/hmrc/helptosavefrontend/controllers/ReminderControllerSpec.scala

```diff
@@ -712,7 +715,7 @@ class ReminderControllerSpec
     }
 
     "should redirect to internal server error page if htsUser update fails in savings reminder  signup page " in {
-      val htsUserForUpdate = HtsUserSchedule(Nino(nino), "email", firstName, lastName, true, Seq(1), LocalDate.now())
+      //val htsUserForUpdate = HtsUserSchedule(Nino(nino), "email", firstName, lastName, true, Seq(1), LocalDate.now())
```

Comment:
Why commented?

Reply:
as was not used, but didn't want to remove incase it was there for a reason.